qemu/tests/balloon_check.py

```diff
@@ -163,10 +183,11 @@ def get_memory_boundary(self, balloon_type=''):
         if self.params.get('os_type') == 'windows':
             logging.info("Get windows miminum balloon value:")
             self.vm.balloon(1)
-            time.sleep(90)
+            balloon_timeout = self.params.get("balloon_timeout", 9600)
```

Comment:
How long do you prefer ?  In my test, it takes more than 300s to balloon 16G memory of windows guest.  I put it long enough in case guest memory is too big.